.github/workflows/build.yaml

```diff
@@ -1,63 +1,125 @@
-name: push
+name: Build
 on:
   push:
     branches:
-      - master
       - main
     tags:
       - v*
 
+env:
+  DEFAULT_TAG_VERSION: "latest"
+  BUILD_PLATFORMS: "linux/amd64,linux/arm64"
+  IMAGE_REGISTRY: "ghcr.io"
+
 jobs:
+  setup:
```

Comment:
I think in future/as a next step we can move this `setup` job into a shared workflow, since it is the same for both push/pull_request workflows.

Reply:
Another good one for an issue and a future implementation within a shared workflow